react/features/app/actions.js

```diff
@@ -14,11 +14,23 @@ import {
 } from './actionTypes';
 import {
     _getRoomAndDomainFromUrlString,
-    _getRouteToRender
+    _getRouteToRender,
+    init
 } from './functions';
 import './reducer';
 
 /**
+ * Temporary solution. Should dispatch actions related to
+ * initial settings of the app like setting log levels,
+ * reading the config parameters from query string etc.
+ *
+ * @returns {Function}
+ */
+export function appInit() {
```

Comment:
@lyubomir And also as far as I understand from the codebase we shouldn't initialize the conference until we move to the `Conference` (in old app we did initialization on the `WelcomePage` but it was senseless because we did it again when user went to the conference because page was reloaded and all app initialization was run again). In new app we're trying to get rid of page reloading because it improves the UX.